### PR TITLE
Release 0.10.0

### PR DESCRIPTION
**Pull request by ksenia-krasheninnikova**
New release for organelles subworkflow

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/genomeassembly/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by ksenia-krasheninnikova** [comment]
@gq1 PR is ready for review. Many thanks.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,81 +3,26 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v0.9.1dev - [date]
```
Comment (gq1) [inline CHANGELOG.md:6]: I think it is supposed you keep the history of change log, not just add the change log for the new release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,81 +3,26 @@
...
-### `Fixed`
-
-### `Dependencies`
-
-### `Deprecated`
-
-## [[0.9.0](https://github.com/sanger-tol/genomeassembly/releases/tag/0.9.0)] - Night Fury - [2023-12-15]
+## [[0.9.1](https://github.com/sanger-tol/genomeassembly/releases/tag/0.9.1)] - Hideous Zippleback - [2024-04-15]
 
 Initial release of sanger-tol/genomeassembly, created with the [nf-core](https://nf-co.re/) template.
```
Comment (gq1) [inline CHANGELOG.md:8]: And then this line can be removed for release 0.9.1 changes.

**Comment by gq1** [comment]
Here is the updated version of release instruction from @muffato. 
https://pipelines.tol.sanger.ac.uk/docs/contributing/pipeline-release
It seems you did a lots of changes for this release, probably not a small number change, but you already on 0.9. Probably it will be ok.

**Comment by muffato** [comment]
I agree. 0.9.1 would be for bug-fixes only. Here you've got real features and differences. This should be 0.10.0

**Comment by ksenia-krasheninnikova** [comment]
Addressed the comments from the code review.
Also, updated the code in order to be able to switch on plastid search in oatk - this was missed in the previous PR.
And updated the json schema for the input yaml file.
I will need to update the release date if there are no other comments.
Ready for re-review @gq1 @muffato

File: CHANGELOG.md [hunk 3]
```diff
@@ -60,6 +72,7 @@ Note, since the pipeline is using Nextflow DSL2, each process will be run with i
 | fastk           |             | f18a4e6d2207539f7b84461daebc54530a9559b0 |
 | freebayes       |             | 1.3.6                                    |
 | gatk4           |             | 4.4.0.0                                  |
+| genescope       |             | 380815c420f50171f9234a0fd1ff426b39829b91 |
```
Comment (gq1) [inline CHANGELOG.md:75]: Is this the new dependency for the new release? Then it should not be here.

Comment (ksenia-krasheninnikova) [reply CHANGELOG.md:75]: It's the old dependency missed in this list in the previous released.

**Review by gq1 — APPROVED** [review]

**Pull request merged**